src/state/apis/swapper/getSwappersApi.ts

```diff
@@ -19,27 +24,32 @@ type GetAvailableSwappersReturn = {
 
 export const getSwappersApi = swapperApi.injectEndpoints({
   endpoints: build => ({
-    getAvailableSwappers: build.query<GetAvailableSwappersReturn, GetSwappersWithQuoteMetadataArgs>(
-      {
-        queryFn: async (args, { getState }) => {
-          const state: State = getState() as unknown as State // ReduxState causes circular dependency
-          const featureFlags = state.preferences.featureFlags
-          const swapperManager = await getSwapperManager(featureFlags)
+    getAvailableSwappers: build.query<GetAvailableSwappersReturn, GetTradeQuoteInput>({
+      queryFn: async (getTradeQuoteInput, { getState }) => {
+        const state: ReduxState = getState() as ReduxState
```

Comment:
👀

Reply:
Confirmed at runtime we have no circular deps here, interesting

Reply:
How... 🤔